domains/andriy.json

```diff
@@ -0,0 +1,11 @@
+ 
+    {
+      "owner": {
+        "username": "andriyandriyan",
+        "email": "andriy.andriyan1@gmail.com"
+      },
+      "record": {
+        "CNAME": "cname.vercel-dns.com"
```

Comment:
Sorry, we don't support Vercel.

Reply:
Hello, i have updated the CNAME to cloudflare